test/Feature/AuthSpec.hs

```diff
@@ -94,9 +89,8 @@ spec = describe "authorization" $ do
   it "hides tables from users with invalid JWT" $ do
     let auth = authHeaderJWT "ey9zdGdyZXN0X3Rlc3RfYXV0aG9yIiwiaWQiOiJqZG9lIn0.y4vZuu1dDdwAl0-S00MCRWRYMlJ5YAMSir6Es6WtWx0"
     request methodGet "/authors_only" [auth] ""
-      `shouldRespondWith` ResponseMatcher {
-          matchBody = Nothing
-        , matchStatus = 401
+      `shouldRespondWith` [json| {"message":"JWT invalid"} |]
```

Comment:
It's kind of awkward that the new version doesn't make it easy to ignore the body and match the other parts. The `Test.Hspec.Wai.Matcher` module even has this internal helper:

```haskell
matchAny :: MatchBody
matchAny = MatchBody (\_ _ -> Nothing)
```

On the other hand maybe asserting the particular output in the body provides better test coverage anyway...